cygnus-twitter/pom.xml

```diff
@@ -0,0 +1,242 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.telefonica.iot</groupId>
+    <artifactId>cygnus-twitter</artifactId>
+    <version>0.1.0_SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>cygnus-twitter</name>
+    <url>http://maven.apache.org</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <!-- Required for testing -->
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+            <version>1.9.5</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.11</version>
+            <scope>test</scope>
+        </dependency>
+        <!-- Required by the extension added by cygnus-common to Apache Flume -->
```

Comment:
None of these dependencies is required by your agent. In any case, they should be added by cygnus-common. Please, double check they are not really used by your software.